test/fixtures/push.json

```diff
@@ -0,0 +1,11 @@
+{
+  "head_commit": {
+    "id": "headcommitsha123"
+  },
+  "repository": {
+    "name": "adapter-azure-functions",
```

Comment:
Just to avoid confusions between azuer and AWS, maybe another dummy repository name can be used here?

Reply:
yeah good point, I'll fix it for now